easybuild/easyconfigs/o/OR-Tools/OR-Tools-7.1-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'OR-Tools'
+version = '7.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://developers.google.com/optimization/'
+description = """Google Optimization Tools (a.k.a., OR-Tools) is an open-source, fast
+and portable software suite for solving combinatorial optimization problems."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s-fix-makefiles.patch']
+checksums = [
+    '6118a931ffcf172c7e2371c3392f82380e9d2a29840daba7caff239ad70d4b44',  # v7.1.tar.gz
+    '35d05c0c57f0691f7570b3560443ed7d933fc97c13dc18c2227e2a251c127414',  # OR-Tools-7.1-fix-makefiles.patch
+]
+
+github_account = 'google'
```

Comment:
This should probably go above `source_urls`?

Doesn't matter w.r.t. functionality of course, just a style thing.